### ISSUE TITLE
Broke Highlights on <> lists that span multiple lines

### ISSUE THREAD

**Issue by zoffixznet** [MadcapJake/language-perl6fe#6]
See image (second  `< ... >` list has a newline between items at some point to fit it on multiple lines):
![bug2](https://cloud.githubusercontent.com/assets/5747918/11793512/0be29ee2-a27a-11e5-9c65-a7e5fe3af684.png)

**Comment by MadcapJake** [comment]
Can those actually be multi-line? I didn't realize they could.

**Comment by zoffixznet** [comment]
Yup. I got that in a module and tests seem to pass :) https://github.com/zoffixznet/perl6-Number-Denominate/blob/master/lib/Number/Denominate.pm6#L63

**Comment by MadcapJake** [comment]
This is gonna be hard, the Perl6 compiler figures it out because it keeps track of what kind of expression you can build where, but since this highlighter only sees the syntax, making this multiline captures whenever any less-than operator is in a line :disappointed:

**Comment by zoffixznet** [comment]
Then close the issue :) it's not tgat important :)

**Comment by MadcapJake** [comment]
I could require that (since it seems idiomatic) in order to be considered a quoteword construct, each delimiter must be immediately followed/preceded by a letter.  I know this isn't actually required but at least it would allow for some multi-line quotewords and it seems that often most people do this anyways.
